From a64f2c4a78cde126cc007b3baa9a0c5c2a7c3d44 Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.liu@freescale.com>
Date: Tue, 8 Jan 2013 15:48:57 +0800
Subject: [PATCH 227/280] ENGR00239212 IPU lib:Support dummy IPU lib for
 MX6DQ/DL

This patch supports dummy IPU lib for MX6DQ/DL Android
since vpu unit test may require that to pass build.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
---
 ipu/Android.mk | 11 +++++++----
 1 file changed, 7 insertions(+), 4 deletions(-)

diff --git a/ipu/Android.mk b/ipu/Android.mk
index 2e33736..cd22893 100644
--- a/ipu/Android.mk
+++ b/ipu/Android.mk
@@ -1,11 +1,15 @@
-ifeq ($(BOARD_SOC_CLASS),IMX5X)
 ifeq ($(HAVE_FSL_IMX_IPU),true)
 LOCAL_PATH := $(call my-dir)
 include $(CLEAR_VARS)
 
+ifeq ($(BOARD_SOC_CLASS),IMX5X)
+LOCAL_SRC_FILES := \
+	mxc_ipu_hl_lib.c \
+	mxc_ipu_lib.c
+else
 LOCAL_SRC_FILES := \
-       mxc_ipu_hl_lib.c \
-       mxc_ipu_lib.c
+	mxc_ipu_hl_lib_dummy.c
+endif
 
 LOCAL_CFLAGS += -DBUILD_FOR_ANDROID
 
@@ -19,4 +23,3 @@ LOCAL_PRELINK_MODULE := false
 LOCAL_MODULE_TAGS := eng
 include $(BUILD_SHARED_LIBRARY)
 endif
-endif
-- 
1.8.0

